app/components/modal/publicCloudProductDelete.tsx

```diff
@@ -48,7 +54,8 @@ export const openPublicCloudProductDeleteModal = createModal<ModalProps, ModalSt
       isError: isDeleteProductError,
       error: deleteProductError,
     } = useMutation({
-      mutationFn: () => deletePublicCloudProduct(product.licencePlate),
+      mutationFn: (requestComment: TypeOf<typeof commentSchema>) =>
```

Comment:
`TypeOf<typeof commentSchema>` is being used in multiple places, please make a type in the schema file